app/Http/Controllers/ApiPagesController.php

```diff
@@ -2,15 +2,84 @@
 
 namespace Suyabay\Http\Controllers;
 
+use Auth;
+use Suyabay\AppInfo;
+use Firebase\JWT\JWT;
+use Suyabay\Http\Requests;
+use Illuminate\Http\Request;
+use Illuminate\Database\QueryException;
+use Suyabay\Http\Controllers\Controller;
+
 class ApiPagesController extends Controller
 {
+    /**
+     * Displays the index page of the API.
+     *
+     * @return \Illuminate\Http\Response
+     */
     public function index()
     {
         return view('api.pages.index');
     }
 
+    /**
+     * Display list of apps the user have created and a link to create new appp.
+     *
+     * @return \Illuminate\Http\Response
+     */
     public function myApp()
     {
         return view('api.pages.myapp');
     }
+    
+    /**
+     * Generate a token for user with passed Id.
+     *
+     * @param int $userId
+     *
+     * @return string
+     */
+    public function generateToken()
+    {
+        $appSecret = getenv('APP_SECRET');
+        $jwtAlgorithm = getenv('JWT_ALGORITHM');
+        $timeIssued = time();
+        $tokenId = base64_encode(getenv('TOKENID'));
+        $token = [
+            'iss'  => 'http://suyabay-staging.herokuapp.com/',
```

Comment:
**Code Quality - Nitpick**
@andela-araimi Can you please comment on Line 49, describing what `iss` means.